src/jrd/scl.epp

```diff
@@ -295,16 +287,17 @@ void SCL_check_create_access(thread_db* tdbb, int type)
 	Jrd::Attachment* const attachment = tdbb->getAttachment();
 
 	// Allow the locksmith any access to database
-	// TDBB_trusted_ddl flag may be set in CREATE PACKAGE node. I suppose
-	// if user has CREATE PACKAGE privilege we should not require his to have CREATE FUNCTION as well
-	if ((tdbb->tdbb_flags & TDBB_trusted_ddl) ||
-			attachment->locksmith(tdbb, SystemPrivilege::MODIFY_ANY_OBJECT_IN_DATABASE))
+	if (attachment->locksmith(tdbb, SystemPrivilege::MODIFY_ANY_OBJECT_IN_DATABASE))
 		return;
 
 	const SecurityClass::flags_t obj_mask = SCL_get_object_mask(type);
 
 	if (!(obj_mask & SCL_create))
+	{
+		const char* name = accTypeNumToStr(type);
+//		ERR_post(Arg::Gds(isc_dyn_no_create_priv) << name);
 		ERR_post(Arg::Gds(isc_dyn_no_priv));
```

Comment:
Forgot to switch to the new constant?

Reply:
Yep. In order to compare test results I changed it back and forgot to return.